### PR TITLE
Adds a search bar so user can sort through comments

### PR DESCRIPTION
**Pull request by cjoshea9**
Adds a search bar allowing user to sort through comments if the comment contains the search term regardless of case.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -45,15 +45,21 @@ function showSlide(n) {
 async function showComments(num=-1) {
   const response = await fetch('/data');
   const comments = await response.json();
+  renderComments(comments, num);
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:48]: Would be cool to either filter on the server, or filter current comments (without making a request) on the client.

Comment (cjoshea9) [reply portfolio/src/main/webapp/script.js:48]: I think I am going to try and filter on the server next and hopefully do a little more advanced filtering.

Comment (cjoshea9) [reply portfolio/src/main/webapp/script.js:48]: @g-harel

Comment (g-harel) [reply portfolio/src/main/webapp/script.js:48]: Nice, is that something you wanna work on in this PR or as a follow-up?

Comment (cjoshea9) [reply portfolio/src/main/webapp/script.js:48]: Probably a follow up since I am going to be moving the logic to the backend.

**Pull request merged**